Makefile

```diff
@@ -7,6 +7,9 @@ clean:
 	find . -name __pycache__ -print0 | xargs -0 rm -rf
 	find . -name "*.pyc" -print0 | xargs -0 rm -rf
 	find . -name "*.egg-info" -print0 | xargs -0 rm -rf
+	rm python/nav/web/static/js/package-lock.json
+	rm core
+	rm -rf python/nav/reports/
```

Comment:
If something is generating files in this directory, something is wrong. The culprit should be found (instead of the symptom being masked with `make clean`).

Perhaps some of the JS stuff is generating reports in the wrong directory (missing an extra `../..` or something)